### PR TITLE
DEV-3891-Front-back subject

### PR DESCRIPTION
**Pull request by Hamzaelkhatri**

### CODE REVIEW

**Review by zainabdnaya — COMMENTED** [review]
In general, everything is good except for the fact You forgot to mention the case of the alphabet `a` and its previous.

**Comment by Hamzaelkhatri** [comment]
@zainabdnaya fixed

**Review by zainabdnaya — COMMENTED** [review]
All Good.

**Review by davhojt — DISMISSED** [review]
Good first effort. Please make the following changes and request another review from me.

**Comment by davhojt** [comment]
@zainabdnaya.

Take a look at my comments on this PR, so that you can challenge if you see similar opportunities 👍🏾

File: subjects/front-back/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+## frontback
+
+### Instructions
+
+- Write a program that prints the previous and next alphabet of a given alphabet.
```
[the remark this answers is not in the document]
Comment (Hamzaelkhatri) [reply subjects/front-back/README.md]: fixed

[the remark this answers is not in the document]
Comment (Hamzaelkhatri) [reply subjects/front-back/README.md]: fixed

**Comment by davhojt** [comment]
@Hamzaelkhatri One more thing:

The directory name needs to lose the `-`. Change the directory name. All of the go exam exercises have nocase filenames (I don't know what they're actually called).

The directory name should be like `frontback` like the title

File: subjects/frontback/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+## frontback
+
+### Instructions
+
+Write a program that prints the previous letter and the next letter of the alphabet.
```
Comment (mohamedLazyBob) [inline subjects/frontback/README.md:5]: LGTM, except that this doesn't mention explicitly only the Latin alphabet!

**Review by mohamedLazyBob — APPROVED** [review]

File: subjects/frontback/README.md (hunk 2 above)
Comment (Hamzaelkhatri) [reply subjects/frontback/README.md]: fixed.

**Review by zainabdnaya — APPROVED** [review]

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: subjects/frontback/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+## frontback
```
[the remark this answers is not in the document]
Comment (Hamzaelkhatri) [reply subjects/frontback/README.md:1]: fixed

**Review by MSilva95 — APPROVED** [review]

**Review by jotapero — APPROVED** [review]
Looks Good :alien:

File: subjects/frontback/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+- If the length of the argument is different from `1`, print a new line `\n`.
+- If the argument is not a letter, print a new line `\n`.
+- There are no letters to print before `a` or after `z`.
+
+### Usage
+
+```console
+$ go run . | cat -e 
+$
+$ go run . a | cat -e
```
Comment (tiagocollot) [inline subjects/frontback/README.md:17]: the letter 'a' should be represented as a string like the following outputs: "a"

File: subjects/frontback/README.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+ab$
+$ go run . "B" | cat -e
+ABC$
+$ go run . "z" | cat -e
+yz$
+$ go run . "Hello World!" | cat -e
```
Comment (tiagocollot) [inline subjects/frontback/README.md:23]: If you try to - go run . "Hello Wordl!", it displays a weird behavior not printing the expected output, I would suggest to remove the '!' from the string in order for it to work properly.

**Review by tiagocollot — CHANGES_REQUESTED** [review]

File: subjects/frontback/README.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+## frontback
+
+### Instructions
+
+Write a program that prints the previous letter and the next letter of the Latin alphabet.
+
+- If the number of arguments is different from `1`, print a new line `\n`.
+- If the length of the argument is different from `1`, print a new line `\n`.
+- If the argument is not a letter, print a new line `\n`.
+- There are no letters to print before `a` or after `z`.
```
Comment (tiagocollot) [inline subjects/frontback/README.md:10]: This is a suggestion: "No letters can be printed before 'a' or after 'z'".

File: subjects/frontback/README.md (hunk 4 above)
Comment (jotapero) [reply subjects/frontback/README.md:17]: fixed

**Pull request merged**